src/io/flutter/run/FlutterRunner.java

```diff
@@ -41,7 +41,9 @@ protected DartUrlResolver getDartUrlResolver(@NotNull final Project project, @No
     return new FlutterUrlResolver(project, contextFileOrDir);
   }
 
-  @Override
+  // TODO(devoncarew): This override is commented out until the next stable rev.
```

Comment:
Alex just released a new Dart plugin w/ this change today - I'll update to that version in the build script.

At runtime, we may want to check for a min. required version of the Dart plugin.